unitary/examples/quantum_chinese_chess/chess.py

```diff
@@ -402,7 +402,18 @@ def apply_move(self, str_to_parse: str) -> None:
             Jump(move_variant)(source_0, target_0)
         elif move_type == MoveType.JUMP:
             Jump(move_variant)(source_0, target_0)
-        # TODO(): apply other move types.
+        elif move_type == MoveType.SLIDE:
```

Comment:
what version of python is this targetting?  With 3.11 this big if/else can be made much nicer with:

```py
match move_type:
    case MoveType.CLASSICAL:
        # code
    case MoveType.JUMP:
        # code
```

I think we'd also get lint checks of not all enum values are checked.